### PR TITLE
Add inkscape package

### PR DESCRIPTION
**Pull request by uberhacker**
What is Inkscape?
Inkscape is professional quality vector graphics software which runs on Windows, Mac OS X and GNU/Linux. It is used by design professionals and hobbyists worldwide, for creating a wide variety of graphics such as illustrations, icons, logos, diagrams, maps and web graphics. Inkscape uses the W3C open standard SVG (Scalable Vector Graphics) as its native format, and is free and open-source software.  See https://inkscape.org/about/.

Tested on:
- [x] aarch64
- [x] armv7l
- [x] i686
- [x] x86_64

### CODE REVIEW

**Comment by JL2210** [comment]
No! Not another image editor!

**Comment by JL2210** [comment]
Look at your i686 binary.

File: packages/inkscape.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  homepage 'https://inkscape.org/'
+  version '0.92.3'
+  source_url 'https://inkscape.org/gallery/item/12187/inkscape-0.92.3.tar.bz2'
+  source_sha256 '063296c05a65d7a92a0f627485b66221487acfc64a24f712eb5237c4bd7816b2'
+
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/inkscape-0.92.3-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/inkscape-0.92.3-chromeos-armv7l.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/inkscape-0.92.3-chromeos-x86_64.tar.xz',
+    i686: 'file:///home/chronos/user/chromebrew/release/i686/inkscape-0.92.3-chromeos-i686.tar.xz'
```
Comment (JL2210) [inline packages/inkscape.rb]: Maybe upload this to Bintray?

Comment (uberhacker) [reply packages/inkscape.rb]: Good catch.  Fixed.

**Comment by uberhacker** [comment]
This is no ordinary image editor.  This is a vector graphics editor.  It gives Adobe Illustrator a run for its money.

**Pull request merged**